src/btree/bt_cursor.c

```diff
@@ -1112,9 +1112,9 @@ __wt_btcur_remove(WT_CURSOR_BTREE *cbt, bool positioned)
         ret = __cursor_row_modify(cbt, WT_UPDATE_TOMBSTONE);
     } else {
         ret = __cursor_col_search(cbt, NULL, NULL);
+        WT_ERR_NOTFOUND_KEEP(ret);
```

Comment:
Can this become `WT_ERR_NOTFOUND_KEEP(__cursor_col_search(cbt, NULL, NULL));`?

Reply:
Done. I also changed other uses that follow the same paradigm.